### PR TITLE
Next.js deps update

### PR DESCRIPTION
**Pull request by kazuma0129**
# Description

The `src/nextjs` dir dependency is out of date and is causing errors when run on node.js v18 or later. This PR fixes this problem by updating the `src/nextjs` dir dependency and explicitly specifying the node.js version.

Fixes N/A

## Type of change

- [x] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Code style update
- [ ] Refactor

If changing the UI of default theme, please provide the before/after screenshot.

# Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes, please describe the impact.

# You have tested in the following environments: (Providing a detailed version will be better.)

- [x] Chrome v119.0.6045.199
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] Android
- [ ] iOS

# Final checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

# Other information

### CODE REVIEW

File: src/nextjs/package.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@line/liff": "2.22.2",
-    "next": "11.1.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@line/liff": "2.24.0",
```
Comment (kazuma0129) [inline src/nextjs/package.json:12]: Although not related to the main purpose of this PR, I have changed to use the latest version.

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+20.18.0
```
Comment (kazuma0129) [inline .nvmrc:1]: The fact that the node.js version was not specified on the project side was another factor in this problem I think. So I specified v20.18.0, which is the current LTS.

File: src/nextjs/package.json [hunk 3]
```diff
@@ -9,13 +9,13 @@
...
+    "next": "15.0.1",
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
   },
   "devDependencies": {
-    "eslint": "7.30.0",
-    "eslint-config-next": "11.0.1"
+    "eslint": "9.13.0",
+    "eslint-config-next": "15.0.1"
```
Comment (odanado) [inline src/nextjs/package.json:19]: The following error occurs when lint is run.

```
$ yarn run lint
yarn run v1.22.21
$ next lint
Failed to load config "next/babel" to extend from.
Referenced from: /path/to/line/line-liff-v2-starter/src/nextjs/.eslintrc
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Could you modify the ESLint configuration file with reference to the current Next.js template?
https://github.com/vercel/next.js/blob/v15.0.1/packages/create-next-app/templates/default/js/eslintrc.json

Comment (kazuma0129) [reply src/nextjs/package.json:19]: I have fixed it. 4191fff

**Pull request merged**